Pipfile

```diff
@@ -30,3 +30,4 @@ scrapy-playwright = "*"
 requests-cache = "*"
 requests = "*"
 pycountry = "*"
+pandas = "*"
```

Comment:
I would prefer not to add a huge dependency like `pandas`, if possible. The filtering you're doing with the CSV can be done with built-in Python code.

Reply:
I'm a little confused as why it's in this PR as it's not used in these spiders?

I'm not against it being included, IIRC @jleedev was thinking about h3 a while ago.

Reply:
@Cj-Malone  i created this function and file for having a complete world wide grid in several sizes, to use it for audi. But it appears after that audi scrapping can be done differently (cf commits). 
I still think a world wide grid in different sizes can be useful (instead of several files for each countries or eu only)
i can move it into another PR if you want

Reply:
@iandees we can do the filtering manually but is it a good practice ? and it might take longer and take more memory. 
we can discuss this in another PR if you want. i'll remove the commits from this one